src/cpp/src/whisper/whisper_feature_extractor.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <cstdint>
+#include <cstring>
+#include <vector>
+
+#include "openvino/genai/visibility.hpp"
+
+namespace ov {
+namespace genai {
+
+class OPENVINO_GENAI_EXPORTS WhisperFeatureExtractor {
```

Comment:
currently, it's not public API => we don't need export here.

Reply:
Removed from public API